codespell.txt

```diff
@@ -1 +1,2 @@
 developpment
+FileTest
```

Comment:
> Yet there is one very peculiar problem: `codespell` wants `file test` instead of `FileTest` (literal name of Ruby class from standard library)

It can be solved by:

```suggestion
filetest
```

Reply:
Thanks. Done.